plots/plot.py

```diff
@@ -30,6 +30,18 @@ def setGlobalDefaults():
 light_red = "#fb9a99"
 dark_red = "#e31a1c"
 
+
+def save(figure, name):
+    # Do not emit a creation date, creator name, or producer. This will make the
+    # content of the pdfs we generate more deterministic.
+    metadata = {'CreationDate': None, 'Creator': None, 'Producer': None}
+
+    figure.savefig(name, metadata=metadata)
```

Comment:
I like to output a line indicating the file saved to, since I can easily copy paste that to view the figure

Reply:
Yes. That would be a great idea. I am currently writing a proposal so don't have headspace to push this, but feel free to add this as another patch/PR. There are plenty such small changes that we want to do.